app/app.ts

```diff
@@ -26,10 +25,19 @@ import VeeValidate from 'vee-validate';
 import ChildWindow from 'components/windows/ChildWindow.vue';
 
 const { ipcRenderer, remote } = electron;
-
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 const isProduction = process.env.NODE_ENV === 'production';
 
+window["obs"] = window['require']('obs-studio-node');
+
+{ // Set up things for IPC
+  debugger;
```

Comment:
debugger

Reply:
Force push didn't go through before creating the PR, it's gone now.